Marlin/src/HAL/STM32/pinsDebug.h

```diff
@@ -108,7 +107,11 @@ const XrefInfo pin_xref[] PROGMEM = {
 /**
  * Translation of routines & variables used by pinsDebug.h
  */
-#define NUMBER_PINS_TOTAL NUM_DIGITAL_PINS
+
+#if PA0 >= NUM_DIGITAL_PINS
```

Comment:
Should this be `A0` (analog 0) instead of `PA0` (port A pin 0) ?

Reply:
Peeking at the platform code, it looks like `NUM_ANALOG_FIRST >= NUM_DIGITAL_PINS` might be more suitable here.